routers/common/repo.go

```diff
@@ -21,7 +22,63 @@ import (
 	"code.gitea.io/gitea/modules/util"
 )
 
-// ServeBlob download a git.Blob
+func setCommonHeaders(ctx *context.Context, name string, data interface{}) error {
+	// Google Chrome dislike commas in filenames, so let's change it to a space
+	name = strings.ReplaceAll(name, ",", " ")
+
+	// Fixme: ctx.Repo.Repository.IsPrivate is nil
```

Comment:
where is this issue coming from?

Reply:
https://github.com/go-gitea/gitea/commit/1bfb0a24d843e10d6d95c4319a84980485e584ed#diff-6da01db79cc03b39079c8ef80e36c1c96cb83b7e0c0c65086d5f84ba79c6ae44R88-R89